docs/content/en/docs/tasks/cluster/manually-renew-certs.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Manually Renew Certificates"
+linkTitle: "Manually Renew Certificates"
+weight: 31
+date: 2021-11-04
+description: >
+  How to renew certificates in etcd and control plane machines
+---
+
+Certificates in external etcd and control plane nodes expire after 1 year. The following table shows all the certificate files:
+
+
+| etcd certs             | control plane certs       |
+|-----------------------|--------------------------|
+| apiserver-etcd-client | apiserver-etcd-client    |
+| ca                    | ca                       |
+| etcdctl-etcd-client   | front-proxy-ca           |
+| peer                  | sa                       |
+| server                | etcd/ca.crt              |
+|                       | apiserver-kubelet-client |
+|                       | apiserver                |
+|                       | front-proxy-client       |
+
+You can renew all certificates in the table except `ca` by following the steps given below, where the commands used in Bottlerocket nodes are more sophisticated.
+
+### External etcd node
+
+If the cluster has external etcd, you need to renew the certificates in etcd nodes first.
+
+{{% alert title="Note" color="primary" %}}
+You can find out whether any external etcd node exists by running the following command:
+```
+kubectl get etcdadmcluster -A
+```
+{{% /alert %}}
+
+1. ssh into the etcd node and run the following commands:
+
+{{< tabpane >}}
+{{< tab header="Normal OS" lang="bash" >}}
```

Comment:
Please change this back to Ubuntu/RHEL. Using "Normal OS" implies that Bottlerocket is not normal.